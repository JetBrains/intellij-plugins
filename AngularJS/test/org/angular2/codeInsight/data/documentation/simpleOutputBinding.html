<app-list-item (twoWay<caret>Change)=""></app-list-item>