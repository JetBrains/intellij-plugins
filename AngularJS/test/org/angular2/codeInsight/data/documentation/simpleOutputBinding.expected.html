<div class='definition'><pre>ListItemComponent.twoWayChange: <span style="color:#000080;">any</span></pre></div><div class='content'><p>twoWayChange comment</div><table class='sections'><tr><td valign='top'><icon src='JavaScriptPsiIcons.FileTypes.TypeScriptFile'/>&nbsp;deps/list-item.component.ts</td></table>